go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/smartcontractkit/chainlink-solana v1.0.3-0.20230216171252-49e2a3db1b5c
 	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20230125162500-4e7ec33e60db
 	github.com/smartcontractkit/libocr v0.0.0-20221209172631-568a30f68407
-	github.com/smartcontractkit/ocr2keepers v0.6.9-0.20230218145934-0474d444ac70
+	github.com/smartcontractkit/ocr2keepers v0.6.10-0.20230220225603-e2327eca8bb1
```

Comment:
I think you also removed an interface function from registry. Clean that up here too?